app/assets/stylesheets/views/_organisations.scss

```diff
@@ -1,24 +1,50 @@
 .organisations {
-  .organisations__section {
+  .organisations-list__section {
     margin-bottom: $gutter * 2;
 
     .department-count {
       @include bold-80;
     }
 
-    .organisations__logo-wrapper {
+    .organisations-list__item {
       list-style-type: none;
-      padding-bottom: $gutter-two-thirds;
-      margin-bottom: $gutter-two-thirds;
+      padding-bottom: $gutter-half;
+      margin-bottom: $gutter-half;
       border-bottom: 1px solid $grey-2;
 
-      .organisation__works-with {
+      .organisation-list__item-title {
+        @include core-19;
+        display: inline-block;
```

Comment:
Where this is a heading tag, this display attribute is necessary. Where it's a link, it isn't.